controlplane/pkg/apis/remote/connection/connection_helpers.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2018 VMware, Inc.
```

Comment:
if the file changed the header should be changed to 2018-2019. @nickolaev correct me if I wrong

Reply:
Fixed : 3db2edb